internal/pkg/config/config.go

```diff
@@ -84,6 +84,16 @@ func (c Config) IsSink(call *ssa.Call) bool {
 	return false
 }
 
+func (c Config) IsSinkFunction(f *ssa.Function) bool {
```

Comment:
go/gocomments#doc-comments

Reply:
go/ links are a feature of your local DNS.  They are opaque to a wider audience.

> All top-level, exported names should have doc comments, as should unexported type or function declarations with unobvious behavior or meaning. See http://golang.org/doc/effective_go.html#commentary for more information about commentary conventions.

Reply:
One thing I've been wondering about: do we still want a comment even if it would be redundant? e.g. here the comment would be `IsSinkFunction determines whether a Function is a Sink`...

Reply:
I'm not dogmatic about it, but it is in the `Effective Go` standard and linters often expect a doc-comment.

Reply:
From the `#commentary` link above:

> Inside a package, any comment immediately preceding a top-level declaration serves as a doc comment for that declaration. Every exported (capitalized) name in a program should have a doc comment.

Reply:
SG. It's not a big deal either way. In this case, besides the redundancy issue I also noticed that most (none?) of the other exported symbols had docs, so I went with the locally observed "standard".